File: exslide02/ex8.py
def main():
    num1 = get_int("Diga um número: ");
    num2 = get_int("Diga outro número: ");
    for i in range(num1 + 1, num2):
         print(i, sep= " ")

def get_int(string):
        number = 0
        try:
            number = int(input(string))
        except:
            print("número inválido")
            get_int(string)
        return number

main()

File: exslide02/ex8.c
#include <stdio.h>

int get_int(char* text);

int main()
{
    int num1, num2;
    num1 = get_int("Diga um número: ");
    num2 = get_int("Diga outro número: ");
    for(int i = num1 + 1; i < num2; i++)
    {
        printf("%d ", i);
    }
    return 0;
}

int get_int(char* text)
{
	int num;
	printf("%s\n", text);
	scanf("%d", &num);
	return num;
}
